paddle/trainer/NewRemoteParameterUpdater.cpp

```diff
@@ -66,28 +69,68 @@ void NewRemoteParameterUpdater::init(
   // from parameter server
   if (paddle_begin_init_params(parameterClient_)) {
     LOG(INFO) << "paddle_begin_init_params start";
+    if (!optimizerConfigNew_.has_optimizer()) {
+      // NOTE: convert V1 OptimizatioinConfig proto to OptimizerConfig if
+      // OptimizerConfig not set. This makes golang pserver compatible with
+      // handy V1 demos. Support SGD only.
+      // TODO: Remove these lines when golang pserver is production ready
```

Comment:
we really need to simplify this mess in the future. `sgdConfigV2 ` , `optimizerConfigNew_ `, almost a nightmare to maintain.